paper/features.py

```diff
@@ -264,11 +355,14 @@ def does_it_matter(raw_bytes, detected_encoding, should_be_encoding):
             y.add_row(
                 [
                     package,
-                    encoding,
                     round(
                         (report[package]['encodings'][encoding]['success'] / (report[package]['encodings'][encoding]['success'] + report[package]['encodings'][encoding]['failure'])) * 100,
                         ndigits=2
                     ),
+                    round(
+                        mean(report[package]['encodings'][encoding]['similarities_in_failures']),
+                        ndigits=3
+                    ) if len(report[package]['encodings'][encoding]['similarities_in_failures']) > 0 else 'N/A',

```

Comment:
This is not nice. Maybe split into multiple lines?

Reply:
This script is not part of charset-normalizer. only for research purpose, it will be improved in due time.